### ISSUE TITLE
Installation failed for PHP 5.6.27

### ISSUE THREAD

**Issue by Vittalks** [renatomarinho/laravel-gitscrum#155]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for doctrine/dbal 2.6.x-dev -> satisfiable by doctrine/dbal[2.6.x-dev].
    - doctrine/dbal 2.6.x-dev requires php ^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.
  Problem 2
    - Installation request for doctrine/dbal dev-master -> satisfiable by doctrine/dbal[dev-master].
    - doctrine/dbal dev-master requires php ^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.
  Problem 3
    - Installation request for sebastian/global-state 2.0.x-dev -> satisfiable by sebastian/global-state[2.0.x-dev].
    - sebastian/global-state 2.0.x-dev requires php ^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.
  Problem 4
    - Installation request for sebastian/global-state dev-master -> satisfiable by sebastian/global-state[dev-master].
    - sebastian/global-state dev-master requires php ^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.
  Problem 5
    - doctrine/dbal 2.6.x-dev requires php ^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.
    - xethron/migrations-generator v2.0.0 requires doctrine/dbal ~2.4 -> satisfiable by doctrine/dbal[2.6.x-dev].
    - Installation request for xethron/migrations-generator v2.0.0 -> satisfiable by xethron/migrations-generator[v2.0.0].

**Comment by willianmano** [comment]
The error stack is clear. 

^7.0 -> your PHP version (5.6.27) does not satisfy that requirement.

You need PHP Version 7+ to run the project.

**Comment by Vittalks** [comment]
Yes, but Installation requirements for Laravel GitScrum application is like below php version given

PHP - Supported Versions: PHP 5.6, PHP 7.0 or HHVM

so that if we tried with php 5.6.* not working
please help me out

**Comment by willianmano** [comment]
Try to use docker

**Comment by Vittalks** [comment]
We can use docker but i changed to php7 than installation done thanks.

but how to run this?

**Comment by lakshmajim** [comment]
Hi @Vittalks ,
You can follow the instructions [here](https://github.com/gitscrum-community-edition/laravel-gitscrum#gitlab)
